docsrc/interactivity.nim

```diff
@@ -27,6 +27,8 @@ nbCode:
   ## Uncomment this line:
   ##script.addToDocAsJs()
 script.addToDocAsJs()
+
+nbText: "The complete code is:"
```

Comment:
ah, no, sorry we cannot do it like this. `nbCodeToJsShowSource()` must come after the block. checking the document the code block is now disappeared (it is showing the code block of `nbText`). let's just remove this

Reply:
Oh right... 🤦 It was part of the nbCodeToJs block. So we should add it to the partial instead then I guess?

Reply:
you are right! we should find the block and then create a new code block with the code content. or in principle for this case we could just specify before the first code block that we will show also the nim code that will be compiled to javascript.

Reply:
> I don't really understand this part.

that part means that we do not change anything in `nbCodeToJsShowSource` and we just add some text before the code blocks saying that the after the first code block, in the second code block we will show the nim source. just a patch and we delay the fix.

 > My understanding is that we throw the "nbJsScriptNimSource" partial out the window completely, and create a new code block in nbCodeToJsShowSource which takes the code from the closest nbCodeToJs block in nb.blocks?

yes this would be my second suggestion. but now that you say it like that, indeed I like better your suggestion of having a message (even a default message) to show in nbCodeToJsShowSource.

Reply:
yep, go ahead!